lib/snippets/push-to-heroku

```diff
@@ -45,15 +45,17 @@ if [[ $? -ne 0 ]]; then
   exit 1;
 fi
 
-MIGRATIONS=`heroku run rake db:migrate:status --app $HEROKU_APP | grep -e '^\s*down'`
+MIGRATION_STATUS=`heroku run rake db:migrate:status --app $HEROKU_APP | grep -e '^\s*down'`
 
-if [[ "${MIGRATIONS}" != "" ]]; then
+if [[ "${MIGRATION_STATUS}" != "" ]]; then
   green "There are pending migrations to apply:"
-  echo $MIGRATIONS
+  echo $MIGRATION_STATUS
```

Comment:
IMO it's more obvious what it actually is (`rake db:migrate:status`) and works better with `POST_MIGRATION_STATUS`